src/main/java/edu/umich/tl/Utils.java

```diff
@@ -64,6 +66,13 @@ public static InputStream getInputStreamFromURL(String fileURL) {
 
 		return in;
 	}
+	public static String dateChopper(String dateAsString) {
```

Comment:
Ok. This is good information. I don't need that date info from the canvas as i will be getting that info from the Mpathways or others. This information is not used anymore So i will take it out